getting-started/CONCEPTS/STREAMING.md

```diff
@@ -48,8 +48,6 @@ The expected sequence of calls is:
 2. `queueTextChunk()`, ..., 
 3. `endStream()`.
 
-Once `endStream()` is called, the stream is considered ended and no further updates can be sent.
```

Comment:
Sorry should have specified Steve, meant to request removing this line instead
"- Streaming is not available in conjunction with AI SDK's function calls yet."